bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/MultiProbeStrategy.scala

```diff
@@ -0,0 +1,70 @@
+package edu.gemini.ags.impl
+
+import edu.gemini.ags.api.AgsMagnitude.MagnitudeTable
+import edu.gemini.ags.api.AgsStrategy.Estimate
+import edu.gemini.ags.api.{AgsAnalysis, AgsMagnitude, AgsStrategy}
+import edu.gemini.catalog.api.CatalogQuery
+import edu.gemini.spModel.ags.AgsStrategyKey
+import edu.gemini.spModel.core.{SingleBand, RBandsList, SiderealTarget}
+import edu.gemini.spModel.guide.{ValidatableGuideProbe, GuideProbe}
+import edu.gemini.spModel.obs.context.ObsContext
+
+import scala.concurrent.Future
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+/**
+ * Represents an AGS strategy consisting of multiple substrategies.
+ *
+ * There are several inherent shortcomings with this:
+ * 1. select requires a position angle, and querying each substrategy may return a number of position angles.
+ *    In the current implementation, the position angle for the first strategy is returned.
```

Comment:
I'm changing this to be more explicit, as you'll see in my next commit in a few minutes.

Currently, the only cases we are interested in are those which involve guide probes where:
1. One of the guide probes is NOT symmetric with respect to rotations around the base / an offset position; and
2. A second guide probe IS symmetric with respect to rotations around the base / an offset position;
Thus, I *think* it makes sense to use the PA as returned by the first guide probe, unless I am possibly misunderstanding something that could come up with respect to offsets. (I am not sure - please tell me if this is the case.)